m4/ax_cpu_freq.m4

```diff
@@ -88,7 +88,7 @@ static float average_MHz(int tries = 2)
     of.close()
 ])],
      [ax_cpu_freq=`cat conftest_cpufreq`; rm -f conftest_cpufreq],
-     [ax_cpu_freq=unknow; rm -f conftest_cpufreq]
+     [ax_cpu_freq=unknown; rm -f conftest_cpufreq]
```

Comment:
I hope this one was not on purpose.

Reply:
I think it's ok to change it. This AC_DEFINE should probably also be a quoted string. But this can be opened in a separate PR.